zenodo/config.py

```diff
@@ -291,7 +291,11 @@ def _(x):
     'github-tokens-refresh': {
         'task': 'invenio_github.tasks.refresh_accounts',
         'schedule': crontab(minute=0, hour=3),
-    }
+    },
+    'openaire-failures-retry': {
```

Comment:
moderate: should this go to the low priority queue? If yes, should we play with the time delta to avoid having yet another bunch of tasks at the same time (thus triggering notifications?)

Reply:
- OpenAIRE direct indexing is actually equivalent to DOI registration in terms of importance, so regular priority is correct in this case.
- Regarding timing, I put every 24h (without a specific time), since there's no specific requirement (it's similar to sitemap generation). But having a specific time (maybe even during working hours) might be better, I'll change it to 09:00am UTC.